duckduckgo/chrome-zeroclickinfo
# Tracker Networks Top Offenders - Pill Version

**Reviewer:** @laurengarcia 



## Description:
Updating top offenders card in the main popup with the new pills UX.

TODO:
- new "not enough data" placeholder
- generic site icons
- popup height TBD

## Steps to test this PR:
1. run make
2. load or reload extension in Chrome or FF
3. after browsing 10 websites, the main popup should show 3 pills with the top offenders icons
4. top offenders subview should be unchanged




## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
